### DISCUSSION TITLE
Ocular Sensor Angle for Alt/Az Mounts

### DISCUSSION THREAD

**Discussion by dts350z** [Stellarium/stellarium#3562] [General]
When you setup an Ocular Sensor and Telescope for Alt/Az, the FOV is displayed correctly on the sky, however the angle indicator, displayed on the FOV, remains fixed at whatever you set it for in the  Sensors dialog (like it would for an EQ mount).

I think this behavior is wrong, for Alt/Az. The angle value needs to be calculated to match the displayed FOV. This is causing problems for mosaic planning tools, as the angle shown in Stellarium doesn't agree with the angle control in the planning tools (they give different orientations of the FOV on the sky).

Can this be fixed?

**Comment by gzotti** [comment]
Which angle indicator? Please make a screenshot and mark what bothers you.

**Comment by dts350z** [comment]
Angle Correct (EQ Mount):

![image](https://github.com/Stellarium/stellarium/assets/5871786/4a196289-b27a-4237-a770-9b517f5e05b9)

Angle Incorrect (Alt/Az Mount):

![image](https://github.com/Stellarium/stellarium/assets/5871786/a30c5b1e-390b-473c-bd3c-d500674cca0c)

Again, the FOV is DISPLAYED at the correct angle, it's just the printed angle that is incorrect.

Is there another place where that angle is shown (correctly)?

**Reply by alex-w** [reply]
Why you think that second image has incorrect angle? Please enable visualization of horizontal coordinates system (Alt-Az) and compare the behavior.

**Reply by 10110111** [reply]
The angle is correct—if taken with respect to the equatorial grid. Of course, you can't expect it to be correct both relative to the equatorial grid and to the alt-azimuthal one.

**Comment by dts350z** [comment]
For the purposes of Mosaic Planning one needs to know the "position angle on the sky" which is in the Equatorial Grid. I was hoping to use Stellarium to give me that angle to then populate it in mosaic planning tools and also warn me if the field rotation was deviating too far in certain mosaic frames, given the time of acquisition.  

Short of trying to plate solve a screen shot of Stellarium, or measuring the angle of the displayed FOV to the Equatorial Grid on the screen, is there a better way? I've been googling for a formula but no luck yet.

I thought if the angle displayed on the FOV always reflected the "Position Angle On the Sky" it would be very convenient.

**Reply by alex-w** [reply]
Top right corner has rotation angle, not a position angle

**Comment by dts350z** [comment]
I guess it's the "rotation angle, in the coordinate system of the telescope".

OK so you are resistant to changing it (to rotation angle in the EQ coordinate system), and the search for some tool or combination of tools for mosaic planning with alt/az mount goes on.

**Reply by 10110111** [reply]
I'm not quite sure what exactly equipment you have and what exactly number you want. Let me guess: telescope roll angle with respect to horizon while having the FoV track the equatorial grid?

**Comment by dts350z** [comment]
I have lots and lots of gear, but I was trying to put together instructions for people to plan mosaics with the ZWO Seastar S50, which is Alt/Az.

The difficulty comes in that all the planning  tools assume Equatorial Mounts, and want to know the rotation angle of the camera (in the EQ Grid). They then display the mosaic "Frames", and give you the RA and Dec Coordinates (and rotation angle) you need to target. e.g. from N.I.N.A. (pulling the target in from Stellarium with one click):   
![image](https://github.com/Stellarium/stellarium/assets/5871786/b774ad15-8f24-4d21-8aae-b24794711382)

where you need to input the Rotation Angle.

I hadn't yet thought about taking the Ra &Dec coordinates back to the Seestar UI, but I'm assuming you can enter RA and DEC for targets, vs. just searching for an object. I got hung up on how to get that needed Rotation Angle, from Stellarium.

**Reply by 10110111** [reply]
From your picture the angle looks correct. But the size of the footprint is smaller in NINA than in Stellarium. Something is wrong with your camera parameters.

**Comment by dts350z** [comment]
Screen shots were from points in time where I may not have had the same parameters in both.  The angles were matched because at that point I had an eq mount configured camera in Stellarium. My point was just to show that the Framing UI wanted a (EQ) rotation input. Everything works, as long as you are using an EQ mount.

**Reply by 10110111** [reply]
Well, can you then specify your conditions exactly, preferably with screenshots that demonstrate what you actually see so that I could understand what's wrong?

**Comment by dts350z** [comment]
OK, when you are using an EQ telescope, in Stellarium, this rotation angle can be used in other tools, expecting rotations in EQ coordinates:
![image](https://github.com/Stellarium/stellarium/assets/5871786/a91318dc-96a5-4614-b79d-5c68b3f0982f)

That means that you can display an ocular FOV in Stellarium, and use it as a planning tool for framing observations, and it will agree with other tools like this:
 
![image](https://github.com/Stellarium/stellarium/assets/5871786/6a1d84a1-009e-44a9-8d91-fcc4a3692bed)

and the FOVs line up.

However, If I use a Alt/Az scope in Stellarium (uncheck Equilateral Mount in Ocular Telescope settings) :

![image](https://github.com/Stellarium/stellarium/assets/5871786/3393d31a-a321-4c85-97da-2db23bf63e29)

I'm told, in this thread, that the Rotation Angle is now in the Azimuthal reference, so I **can't** use it in other tools (as they are expecting EQ). If I do:

![image](https://github.com/Stellarium/stellarium/assets/5871786/ff7ec8aa-a670-41ef-b1b6-c5878396d57a)

You can see that the FOVs are at different angles, and I'd have to measure the angle, between the FOV frame and the EQ gridlines in Stellarium, , to know what angle to put in the other tools.  

So, it would be convenient if there was a way to get the Rotation Angle in the EQ frame of reference, even when you have  Equilateral Mount in Ocular Telescope settings unchecked.  Maybe it would need to be an option, or a way to see both, to also keep the current behaviour?

Thanks for taking a look.

**Reply by gzotti** [reply]
Stellarium's behaviour seems to be what all developers have intended to provide. The rotation angle is relative to the current mount system, be it equatorial (not "equilateral") or azimuthal. Probably because in the last 120 years (or so) mosaics would have been planned in EQ mode, so this is hard-coded in our brains. It seems you just need the parallactic angle which is given in the top-left info for all extended objects (DSO, planets).

**Reply by alex-w** [reply]
All screenshots from N.I.N.A. are showing objects in equatorial coordinates.

**Comment by dts350z** [comment]
That can work, but I need to flip the sign. By inspection:

![image](https://github.com/Stellarium/stellarium/assets/5871786/0ca20741-63bb-4acc-a3c8-dd9a23251418)

Thanks!

**Reply by gzotti** [reply]
OK. We might be able to overfreight the GUI and label corners further by optionally adding parallactic angle of image centre (or the equivalent image rotation against E-Q-U-A-T-O-R-I-A-L coordinates) in azimuthal mode.
